larreco/Calorimetry/GnocchiCalorimetry_module.cc

```diff
@@ -78,6 +79,10 @@ namespace calo {
       fhicl::Atom<std::string> T0ModuleLabel{Name("T0ModuleLabel"),
                                              Comment("Module label for T0 time producer."),
                                              ""};
+      fhicl::Atom<std::string> PFPModuleLabel{
+        Name("PFPModuleLabel"),
+        Comment("Module label for PFP producer. To be used to associate T0 with tracks."),
+        ""};
```

Comment:
I understand why a default value would be useful here (as its an optional input label) but it would be good to consider removing it, and relying on fcl to set up the empty string.  Default values can hide fcl file mistakes, where a job silently runs and picks up the default value from the c++ definition rather than notifying the user that it could not find the parameter in the fcl file.

Reply:
I like this. We clearly state it in 2 places (the code and the fcl) what it does, and remove the default so that it cleanly but loudly ends and tells the user what's wrong. And if they copy the fcl block it will also let them know what's going on in the comment. 

@vpec0 Agree to change it?